appveyor.yml

```diff
@@ -1,10 +1,11 @@
-os: Visual Studio 2015
+image:  # http://www.appveyor.com/docs/appveyor-yml
+  - Visual Studio 2022
 
 environment:
-  matrix:
-    - nodejs_version: 10
-    - nodejs_version: 12
-    - nodejs_version: 14
+  matrix:  # https://github.com/nodejs/release#release-schedule
+    - nodejs_version: 16
+    - nodejs_version: 18
+    - nodejs_version: 20
```

Comment:
```suggestion
    - nodejs_version: 20
    - nodejs_version: 21
```

Reply:
i made this change, but i'm not sure it is corrent since it seemed only lts releases were listed here before

Reply:
The pr title says 21 so let’s test 21.